cpp/src/gandiva/function_registry.cc

```diff
@@ -453,6 +274,16 @@ FunctionRegistry::SignatureMap FunctionRegistry::InitPCMap() {
     // printf("%s -> %s\n", entry->signature().ToString().c_str(),
     //      entry->pc_name().c_str());
   }
+
+  FunctionRegistryBin::SignatureMap& binaryFnMap =
+      FunctionRegistryBin::GetBinaryFnSignature();
```

Comment:
might be more optimal to pass the map as arg to Get*FnSignature() and update the same inside the function? would avoid double copy..

Reply:
Refactored this part of the code, so map pointer is no longer needed.